api/src/main/java/io/strimzi/api/kafka/model/KafkaRebalanceSpec.java

```diff
@@ -25,6 +26,10 @@
 public class KafkaRebalanceSpec extends Spec {
     private static final long serialVersionUID = 1L;
 
+    // rebalancing modes
+    private KafkaRebalanceMode mode;
```

Comment:
Given the default is full-rebalance, should it be set as default here as well? (and add the NON_DEFULT include annotation)